src/components/QuickOpenModal.js

```diff
@@ -383,12 +383,14 @@ export class QuickOpenModal extends Component<Props, State> {
             expanded && items[selectedIndex] ? items[selectedIndex].id : ""
           }
         />
-        {!symbols ||
+        {this.isSymbolSearch() &&
```

Comment:
This is becoming hard to read, how about we move this logic to its own function?

And run `jest -u` to update the tests :)